### PR TITLE
Spring boot 3.0

### PR DESCRIPTION
**Pull request by steklopod**
Need to be rechecked

### CODE REVIEW

**Comment by bnasslahsen** [comment]
@steklopod

Thank you for your contribution.
You are breaking the compatibility with previous spring-boot versions.
You can try handling the compatibility with previous spring-boot versions so we can merge it.

File: build.gradle.kts [hunk 1]
```diff
@@ -1,15 +1,16 @@
 plugins {
 	`java-gradle-plugin`
-	id("com.gradle.plugin-publish") version "0.14.0"
-	id("org.sonarqube") version "3.1.1"
-	kotlin("jvm") version "1.4.31"
 	`maven-publish`
-	id("com.github.ben-manes.versions") version "0.38.0"
-	id("io.gitlab.arturbosch.detekt") version "1.16.0"
+	kotlin("jvm") version "1.8.0-Beta"
```
Comment (gotson) [inline build.gradle.kts:4]: Spring Boot 3 uses Kotlin 1.7 AFAIK

**Comment by faha1Ex** [comment]
> @steklopod
> 
> Thank you for your contribution. You are breaking the compatibility with previous spring-boot versions. You can try handling the compatibility with previous spring-boot versions so we can merge it.

Just an idea: Would it maybe make sense to start a version 2.0.0 for this, so that 2.x can be used for Spring Boot 3.0 compatibility and 1.x for previous versions? Of course I am aware that this would mean more maintainability effort regarding future pull requests.

**Comment by bnasslahsen** [comment]
@steklopod,

Not really.
There not too much contributors for the plugin to support parallel version.
Additionnally the sponsoring for it is too low. 
So.there will not be a lot of enhancements on it.

The best is keeping things simple to garantee the long term maintenance

**Pull request closed without merging**